src/commands/watch.ts

```diff
@@ -80,6 +80,14 @@ export function _makeCurrentMigrationRunner(
 
             // 4: if different
             if (!migrationsAreEquivalent) {
+              if (parsedSettings.beforeCurrent) {
+                await executeActions(
+                  parsedSettings,
+                  shadow,
+                  parsedSettings.beforeCurrent,
+                );
+              }
```

Comment:
```suggestion
              await executeActions(
                parsedSettings,
                shadow,
                parsedSettings.beforeCurrent,
              );
```

I don't think this `if` is necessary - `executeActions` handles that for us.

Reply:
Also it will never be falsy, as can be seen from the types.